From 97a51da823e0ecff997b7f1aa410e65ed52e34c0 Mon Sep 17 00:00:00 2001
From: garwedgess <garethwilliams21@gmail.com>
Date: Mon, 13 Apr 2015 19:07:04 +0100
Subject: [PATCH] Change path for i9300 hardware from archi

---
 media/libstagefright/Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/media/libstagefright/Android.mk b/media/libstagefright/Android.mk
index 4a80f18..fadf3ab 100644
--- a/media/libstagefright/Android.mk
+++ b/media/libstagefright/Android.mk
@@ -205,8 +205,8 @@ LOCAL_CFLAGS += -DUSE_SAMSUNG_COLORFORMAT
 
 # Include native color format header path
 LOCAL_C_INCLUDES += \
-	$(TOP)/hardware/samsung/exynos4/hal/include \
-	$(TOP)/hardware/samsung/exynos4/include
+	$(TOP)/hardware/i9300/exynos4/hal/include \
+	$(TOP)/hardware/i9300/exynos4/include
 
 endif
 
-- 
1.9.1

