From e08be9c64b643bab9139a1cd5f549320ecfed6fc Mon Sep 17 00:00:00 2001
From: "R. Andrew Ohana" <andrew.ohana@gmail.com>
Date: Tue, 13 Jan 2015 12:27:59 +0100
Subject: [PATCH] wifi.c: allow devices to specify driver delay

---
 wifi/Android.mk | 3 +++
 wifi/wifi.c     | 6 +++---
 2 files changed, 6 insertions(+), 3 deletions(-)

diff --git a/wifi/Android.mk b/wifi/Android.mk
index ad9f2c9..53ebfe7 100644
--- a/wifi/Android.mk
+++ b/wifi/Android.mk
@@ -15,6 +15,9 @@ endif
 ifdef WIFI_FIRMWARE_LOADER
 LOCAL_CFLAGS += -DWIFI_FIRMWARE_LOADER=\"$(WIFI_FIRMWARE_LOADER)\"
 endif
+ifdef WIFI_DRIVER_LOADER_DELAY
+LOCAL_CFLAGS += -DWIFI_DRIVER_LOADER_DELAY=$(WIFI_DRIVER_LOADER_DELAY)
+endif
 ifdef WIFI_DRIVER_FW_PATH_STA
 LOCAL_CFLAGS += -DWIFI_DRIVER_FW_PATH_STA=\"$(WIFI_DRIVER_FW_PATH_STA)\"
 endif
diff --git a/wifi/wifi.c b/wifi/wifi.c
index 5a19f5b..abd0fcb 100644
--- a/wifi/wifi.c
+++ b/wifi/wifi.c
@@ -130,8 +130,6 @@ static const char EXT_MODULE_PATH[] = WIFI_EXT_MODULE_PATH;
 #define WIFI_DRIVER_FW_PATH_PARAM	"/sys/module/wlan/parameters/fwpath"
 #endif
 
-#define WIFI_DRIVER_LOADER_DELAY	1000000
-
 static const char IFACE_DIR[]           = "/data/system/wpa_supplicant";
 #ifdef WIFI_DRIVER_MODULE_PATH
 static const char DRIVER_MODULE_NAME[]  = WIFI_DRIVER_MODULE_NAME;
@@ -327,7 +325,9 @@ int wifi_load_driver()
     }
 
     if (strcmp(FIRMWARE_LOADER,"") == 0) {
-        /* usleep(WIFI_DRIVER_LOADER_DELAY); */
+#ifdef WIFI_DRIVER_LOADER_DELAY
+        usleep(WIFI_DRIVER_LOADER_DELAY);
+#endif
         property_set(DRIVER_PROP_NAME, "ok");
     }
     else {
-- 
2.1.1


